src/content/en/fundamentals/performance/rail.md

```diff
@@ -78,18 +83,24 @@ For actions that take longer than 500ms to complete, always provide feedback.
 
 Success: Respond to user's touchmoves and scrolling in under 16ms.
 
-## Animation: render frames every 16ms
+## Animation: execute animation code in under 10ms
 
 Animations aren't trivial actions that web apps can opt into.
 For example, scrolling and touchmoves are types of animation.
```

Comment:
FYI I didn't write this article, just updating it. My guess here is something along the lines of "When you hear animations, you may think this only applies to fancy effects. But scrolling and touchmoves are animations"